### DISCUSSION TITLE
Set default global border color for all elements

### DISCUSSION THREAD

**Discussion by easwee** [mantinedev/mantine#6515] [Q&A] [answered]
In v7 I can't find what is the proper way to set a default border color for all elements.

I want AppShell, Paper, Input etc.. to have the same default border color - let's say "red". How to achieve this in createTheme()?

**Comment by rtivital** [comment, marked as answer]
You can apply styles to components with Styles API – https://mantine.dev/styles/styles-api/#classnames-in-themecomponents

**Reply by easwee** [reply]
Thx for the answer - will make it work!

Would be handy to have this in createTheme - something like `defaultBorderColor` or something, since the same gray border seems to be reused through entire mantine ui.
